cmd/core-common-config-bootstrapper/res/configuration.yaml

```diff
@@ -42,7 +42,7 @@ all-services:
     Port: 8500
     Type: "consul"
   
-  Databases:
+  Database:
     Host: "localhost"
     Port: 6379
     Timeout: 5000
```

Comment:
Please remove the topics below as they are specific to Core Data and should be left in Core Data's private config

Reply:
done